code/parse/parselo.cpp

```diff
@@ -667,7 +667,6 @@ int required_string_either(const char *str1, const char *str2)
 	}
 
 	nprintf(("Error", "Error: Unable to find either required token [%s] or [%s]\n", str1, str2));
-	Warning(LOCATION, "Error: Unable to find either required token [%s] or [%s]\n", str1, str2);
```

Comment:
You removed a warning without adding an equivalent `error_display()`; why?

Reply:
There is an `error_display` a few lines above this so this seems to be redundant.

Reply:
The exact same statement is true of `required_string()` above.

Reply:
Indeed, fixed that now.

Reply:
Now you've just got the inconsistency of removing the `nprintf()` call in one but not the other.

Reply:
Not anymore.